### PR TITLE
Synapse Infra Metrics Workbook

### PR DESCRIPTION
**Pull request by felipepessoto**
Creating workbook with some of the basic infrastructure metrics, like CPU and Memory
Removing preview flag from streaming workbook

image

## PR Checklist

* [X] Explain your changes, so people looking at the PR know *what* and *why*, the code changes are the *how*.

### If adding or updating templates:
* [X] post a screenshot of templates and/or gallery changes
* [X] ensure your template has a corresponding gallery entry in the gallery folder
* [X] If you are adding a new template, add your team and template/gallery file(s) to the CODEOWNERS file. CODEOWNERS entries should be teams, not individuals
* [X] ensure all steps have meaningful names
* [X] ensure all parameters and grid columns have display names set so they can be localized
* [X] ensure that parameters id values are unique __or they will fail PR validation__ (parameter ids are used for localization)
* [X] ensure that steps names are unique __or they will fail PR validation__ (step names are used for localization)
* [X] grep `/subscription/` and ensure that your parameters don't have any hardcoded resourceIds __or they will fail PR validation__
* [X] remove `fallbackResourceIds` and `fromTemplateId` fields from your template workbook __or they will fail PR validation__

### CODE REVIEW

**Comment by felipepessoto** [comment]
+ @gardnerjr to help reviewing the Azure Monitor.json

File: gallery/workbook/Azure Monitor.json [hunk 1]
```diff
@@ -74,10 +74,15 @@
...
 					"author": "Microsoft",
-					"name": "Streaming Metrics",
+					"name": "Infrastructure Metrics",
 					"isPreview": true
+				},
+				{
+					"id": "Workbooks/Synapse/Streaming",
+					"author": "Microsoft",
+					"name": "Streaming Metrics"
 				}
```
Comment (yeskarthik) [inline gallery/workbook/Azure Monitor.json:86]: did we remove isPreview:true , is that intentional?

File: gallery/workbook/Azure Monitor.json [hunk 2]
```diff
@@ -74,10 +74,15 @@
 			"name": "Synapse",
 			"templates": [
 				{
-					"id": "Workbooks/Synapse/Streaming",
+					"id": "Workbooks/Synapse/Infrastructure",
```
Comment (yeskarthik) [inline gallery/workbook/Azure Monitor.json:77]: lets be more specific in calling in Synapse/Spark/Infrastructure.. and name is Spark Infrastructure Metrics

**Review by yeskarthik — APPROVED** [review]
Looks good except the naming scheme

File: gallery/workbook/Azure Monitor.json (hunk 2 above)
Comment (felipepessoto) [reply gallery/workbook/Azure Monitor.json]: Thanks, I renamed them. Also removed the 'Metrics" suffix, seems redundant and other workbooks don't have that suffix as well

**Comment by felipepessoto** [comment]
Thanks Karthik.
@microsoft/applicationinsights-devtools, to review the gallery changes please.

**Comment by felipepessoto** [comment]
@microsoft/applicationinsights-devtools ping

**Review by gardnerjr — APPROVED** [review]

**Pull request merged**